tests/containers/validate_btrfs.pm

```diff
@@ -65,15 +66,14 @@ sub _test_btrfs_device_mgmt {
     script_run("df -h");
     # Determine the remaining size of /var
     my $var_free = script_output('df 2>/dev/null | grep /var | awk \'{print $4;}\'');
-    # /var is using its own partition. Create file in the container enough to fill up the partition up to 99%
-    my $fill = int($var_free * 0.99 * 1024);    # df returns the size in KiB
+    # Create file in the container enough to fill the "/var" partition (where the container is located) up to 90%
+    # Note: Don't fill it up more than 90% otherwise `btrfs balance` might stumble.
+    my $fill = int($var_free * 1024 * 0.90);    # df returns the size in KiB
```

Comment:
i will try to explain what was the scenario and why this line was there.
One of the advantages with btrfs is that we can add additional disk space if we need to.
the idea here was that i wanted to simulate a disk that it cant be operate. Filling up the disk, such as it cannot pull another image. Then add `/dev/vdb` and try again, which it should solve the problem.

having `$var_free * 1024 * 0.90` the logic should break. the disk will never fail as we do not use such a big image which it should make impossible to pull it and such test the addition of the extra disk. 

And that was the purpose of the https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12662/files#diff-be0e7754c72dde534df2a385f979c80230bc691216d84c089694db663f6fdfefL76

Reply:
Ah ok, let me see if I can fix this then.